packages/components/pie-thumbnail/src/thumbnail.scss

```
@@ -16,6 +17,34 @@
     --thumbnail-border-color: var(--dt-color-border-default);
   }
 
+  &.c-thumbnail--disabled {
+    opacity: 0.5;
+  }
+
+  &.c-thumbnail--background-subtle {
+    --thumbnail-bg-color: var(--dt-color-container-subtle);
+  }
+
+  &.c-thumbnail--background-strong {
+    --thumbnail-bg-color: var(--dt-color-container-strong);
+  }
+
+  &.c-thumbnail--background-dark {
+    --thumbnail-bg-color: var(--dt-color-container-dark);
+  }
+
+  &.c-thumbnail--background-inverse {
+    --thumbnail-bg-color: var(--dt-color-container-inverse);
+  }
+
+  &.c-thumbnail--background-inverse-alternative {
```

To fit with our CSS namign scheme:

```suggestion
  &.c-thumbnail--background-inverseAlt {
```